### ISSUE TITLE
top-left indentation is lost with Viewer or as_html

### ISSUE THREAD

**Issue by anajens** [Roche/rtables#204]
Following fix for #194, this is still an issue.

The nice indentation in the header is lost when displaying rtables in the Viewer or as html: 
![image](https://user-images.githubusercontent.com/43623360/117355538-b58e9780-ae80-11eb-87b6-ea4121683339.png)

![image](https://user-images.githubusercontent.com/43623360/117355565-c17a5980-ae80-11eb-9e4c-7386571b106d.png)

_Originally posted by @anajens in https://github.com/Roche/rtables/issues/194#issuecomment-833806000_

**Comment by gmbecker** [comment]
@anajens are you still seeing this issue with the fix for #194 in place?

**Comment by anajens** [comment]
Just to add an example which is probably related:

```r

tbl <- basic_table() %>%
  split_rows_by("ARMCD", label_pos = "topleft", split_label = "     XXX") %>%
  analyze("AGE") %>%
  build_table(ex_adsl)
```

Print looks OK
```
     XXX   all obs
------------------
ARM A             
  Mean      33.77 
ARM B             
  Mean      35.43 
ARM C             
  Mean      35.43 
```

In html spaces are ignored:
![image](https://user-images.githubusercontent.com/43623360/119730085-54d0fa00-be43-11eb-9b98-45c11382ded2.png)

This is not a new issue.  I checked back in the Dec 2020 release and the header indentation was not working there either.

**Comment by gmbecker** [comment]
Its not header information, per se but rather top_left material
specifically, but yes I believe its been present since I put top left
functionality in in the first place.

**Comment by anajens** [comment]
It was not too noticeable in tables with 2 levels in the top-left. But with more levels it does make it difficult to read.

**Comment by gmbecker** [comment]
Apologies for it not being tagged (or mentioned in the NEWS file) but this is fixed in version `0.3.7.0007` as well. Please confirm that on your end and if not feel free to reopen @anajens.

**Comment by anajens** [comment]
Hi @gmbecker any chance the top-left header indentation can be aligned with the table row labels?

For example, the second split variable row labels (eg CHN) now seems to be aligned with the third split variable top-left header (Biomarker).
![image](https://user-images.githubusercontent.com/43623360/120408339-d087e680-c31c-11eb-9352-81f1db959553.png)
